src/Commands/Admin/GetTenantInfo.cs

```diff
@@ -7,6 +8,7 @@
 namespace PnP.PowerShell.Commands.Admin
 {
     [Cmdlet(VerbsCommon.Get, "PnPTenantInfo")]
+    [RequiredApiApplicationPermissions("graph/CrossTenantInformation.ReadBasic.All")]
```

Comment:
I think we should also add `RequiredApiDelegatedPermissions("graph/CrossTenantInformation.ReadBasic.All")` here since it works with delegated as well if available. What do you think ?

Reply:
Agree with you, Changed the attribute to `RequiredApiDelegatedOrApplicationPermissions`